### DISCUSSION TITLE
请问下还会添加安卓支持吗

### DISCUSSION THREAD

**Discussion by jacobax** [LufsX/rime#1] [General]
感谢分享

**Comment by LufsX** [comment]
关于同文输入法嘛，我手机上不能正常使用，CI 版也试过了，不知道啥原因，应该不会支持了。

不过欢迎贡献哈~

（抱歉，现在才看到这个 Discussion，都忘记有这玩意了🤣）

**Comment by LufsX** [comment]
已支持～

#3
